### PR TITLE
QMCPACK Update March 2020 - Part 2

### PR DESCRIPTION
**Pull request by naromero77**
- Add support for ppconvert
- Fix broken install method
- Fix broken setup_run_environment method so that nexus will work
- General clean-up, e.g. intel-mkl conditional statement

### CODE REVIEW

File: var/spack/repos/builtin/packages/qmcpack/package.py [hunk 1]
```diff
@@ -323,79 +336,55 @@ def cmake_args(self):
         # Next two environment variables were introduced in QMCPACK 3.5.0
         # Prior to v3.5.0, these lines should be benign but CMake
         # may issue a warning.
-        if 'intel-mkl' in spec:
+        if '^intel-mkl' in spec:
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/qmcpack/package.py]:
```suggestion
        if '^mkl' in spec:
```
Both `intel-mkl` and `intel-parallel-studio+mkl` provide the `mkl` virtual dependency.

File: var/spack/repos/builtin/packages/qmcpack/package.py [hunk 2]
```diff
@@ -323,79 +336,55 @@ def cmake_args(self):
...
 
-    @when('@3.6.0:')
-    def setup_run_environment(self, env):
-        """Set-up runtime environment for QMCPACK.
-        Set PYTHONPATH for basic analysis scripts and for Nexus. Binaries
-        are in the  'prefix' directory instead of 'prefix.bin' which is
-        not set by the default module environment"""
-        env.prepend_path('PATH', self.prefix)
-        env.prepend_path('PYTHONPATH', self.prefix)
+        env.prepend_path('PATH', self.prefix.nexus + '/bin')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/qmcpack/package.py]:
```suggestion
        env.prepend_path('PATH', self.prefix.nexus.bin)
```

File: var/spack/repos/builtin/packages/qmcpack/package.py [hunk 3]
```diff
@@ -323,79 +336,55 @@ def cmake_args(self):
...
+        env.prepend_path('PYTHONPATH', self.prefix.nexus + '/lib')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/qmcpack/package.py]:
```suggestion
        env.prepend_path('PYTHONPATH', self.prefix.nexus.lib)
```

File: var/spack/repos/builtin/packages/qmcpack/package.py (hunk 2 above)
Comment (naromero77) [reply var/spack/repos/builtin/packages/qmcpack/package.py]: Will change

File: var/spack/repos/builtin/packages/qmcpack/package.py (hunk 3 above)
Comment (naromero77) [reply var/spack/repos/builtin/packages/qmcpack/package.py]: Will change

File: var/spack/repos/builtin/packages/qmcpack/package.py (hunk 1 above)
Comment (naromero77) [reply var/spack/repos/builtin/packages/qmcpack/package.py]: A similar change was implemented in QE, but seems to be causing a problem. I just filed an issue about it:
https://github.com/spack/spack/issues/15613

**Comment by naromero77** [comment]
@adamjstewart Can we merge this in soon?

**Comment by alalazo** [comment]
@naromero77 Ping me in a couple of days if you got no review, please.

**Comment by naromero77** [comment]
Let me know if there is any additional changes or clarifications needed before this can be merged in.

**Pull request merged**